tests/wallet/sync/test_wallet_sync.py

```diff
@@ -53,7 +54,10 @@ async def get_nft_count(wallet: NFTWallet) -> int:
 
 class TestWalletSync:
     @pytest.mark.asyncio
-    async def test_request_block_headers(self, simulator_and_wallet, default_1000_blocks):
+    async def test_request_block_headers(self, simulator_and_wallet, default_1000_blocks, consensus_mode: Mode):
+        if consensus_mode != Mode.PLAIN:
```

Comment:
this will still wait for all fixtures to be initialized which takes most time, no?
can you access mode in fixures and skip test that way?

Reply:
ideally, it would be possible to use `@pytest.mark.skipif()` and have it depend on one of the fixtures (`consensus_mode`). However, this appears to be an open issue with pytest: https://github.com/pytest-dev/pytest/issues/7395